script/post-version

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+echo "v$npm_package_version\n" > .VERSION_ANNOTATION
+cat .VERSION_CHANGES >> .VERSION_ANNOTATION
+git tag -a v$npm_package_version -e -f -F .VERSION_ANNOTATION
+rm .VERSION_ANNOTATION .VERSION_CHANGES
```

Comment:
> I also think this could be simplified and we actually don't need another temporary file here:

I tried that as well first, but sadly that does not work, as git seems to not except input on stdin. 

> --force

The original `yarn version` command already wrote it's own non annotated tag. There is a way to pass in an additional argument to `yarn version`, so that it does not create a tag at all. 
But I haven't found out how to change `yarn version` to use this argument by default (will check that again)

> --edit

This allows us to write something on top of the automated commit list as a manual release summary (like breaking changes, etc) and also to throw out unwanted commit msgs

> a file header

👍  would be great, will add that.

Reply:
From `man git-tag`:

```
       -F <file>, --file=<file>
           Take the tag message from the given file. Use - to read the message
           from the standard input. Implies -a if none of -a, -s, or -u
           <keyid> is given.
```